homeassistant/components/leviosa_shades/manifest.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "leviosa_shades",
+  "name": "Leviosa Motor Shades",
+  "codeowners": ["@greg-ha-1990"],
+  "config_flow": true,
+  "dependencies": ["cover"],
+  "documentation": "https://www.home-assistant.io/integrations/leviosa_shades",
```

```suggestion
  "documentation": "https://www.home-assistant.io/integrations/leviosa_shades",
   "integration_type": "hub",
```

See https://developers.home-assistant.io/docs/creating_integration_manifest/#integration-type. Have a look if hub or device is best in your situation.

---

Suggestion accepted, thanks.